src/client/pages/Debugger/components/Offer.tsx

```diff
@@ -24,63 +38,119 @@ enum QuoteType {
   SwedishHouse = 'swedish-house',
 }
 
-export const Offer: React.FC = () => {
-  const [getQuote, { data, refetch }] = useQuoteLazyQuery()
+type QuoteData = {
+  label: string
+  value: QuoteType
+  initialFormValues?: Record<string, unknown>
+}
+
+type QuotesByMarket = Record<Market, QuoteData[]>
+
+const quotesByMarket: QuotesByMarket = {
+  DK: [
+    {
+      label: 'Danish Home',
+      value: QuoteType.DanishHome,
+      initialFormValues: initialDkHomeValues,
+    },
+  ],
+  NO: [
+    {
+      label: 'Norwegian Home',
+      value: QuoteType.NorwegianHome,
+      initialFormValues: initialNoHomeValues,
+    },
+    {
+      label: 'Norwegian Travel',
+      value: QuoteType.NorwegianTravel,
+      initialFormValues: initialNoTravelValues,
+    },
+  ],
+  SE: [
+    {
+      label: 'Swedish Apartment',
+      value: QuoteType.SwedishApartment,
+      initialFormValues: initialSeApartmentValues,
+    },
+    {
+      label: 'Swedish House',
+      value: QuoteType.SwedishHouse,
+    },
+  ],
+}
+
+const getCurrentAvailableQuoteData = (
+  currentMarket: Market,
+  currentQuoteType: QuoteType,
+) => {
+  const marketQuoteTypes = quotesByMarket[currentMarket]
+  const currentQuoteTypeData = marketQuoteTypes.find(
+    ({ value }) => value === currentQuoteType,
+  )
+  return currentQuoteTypeData
+}
+
+export const Offer: React.FC<OfferProps> = ({ sessionToken }) => {
   const [quoteId, setQuoteId] = useState<string>('') // TODO handle multiple quotes
+  const [getQuote, { data, refetch }] = useQuoteLazyQuery()
+  const [hasQuoteCreatingError, setHasQuoteCreatingError] = useState(false)
   const storageState = useStorage()
   const currentLocale = useCurrentLocale()
   const localeIsoCode = getLocaleIsoCode(currentLocale)
   const currentMarket = useMarket()
 
-  const quotesByMarket = {
-    DK: [
-      {
-        label: 'Danish Home',
-        value: QuoteType.DanishHome,
-      },
-    ],
-    NO: [
-      {
-        label: 'Norwegian Home',
-        value: QuoteType.NorwegianHome,
-      },
-      {
-        label: 'Norwegian Travel',
-        value: QuoteType.NorwegianTravel,
-      },
-    ],
-    SE: [
-      {
-        label: 'Swedish Apartment',
-        value: QuoteType.SwedishApartment,
-      },
-      {
-        label: 'Swedish House',
-        value: QuoteType.SwedishHouse,
-      },
-    ],
-  }
-
   const [quoteType, setQuoteType] = useState(
     quotesByMarket[currentMarket][0].value,
   )
 
+  const handleSubmit = async (
+    values: Values,
+    storage: Record<string, unknown>,
+  ) => {
+    if (sessionToken) {
+      await createQuote(
+        storage,
+        localeIsoCode,
+      )({
+        input: {
+          ...values,
+          id: quoteId,
+          currentInsurer: values.currentInsurer || undefined,
+          // @ts-ignore
+          startDate: values.startDate || undefined,
+        },
+      }).catch(() => setHasQuoteCreatingError(true))
```

Comment:
Now that we get a somewhat better error messages, an idea would be to print it out to give you some clue on the error
```suggestion
      }).catch((e) => {
         setQuoteCreatingError(e.message)
       })
```

Reply:
Yep, that might be a good idea. 👍 I think I tested it but I don't remember why I removed it haha!

Reply:
Have added the error message!